src/App.js

```diff
@@ -1,29 +1,22 @@
-import logo from './logo.svg';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React, { Component } from 'react';
+// eslint-disable-next-line import/extensions
```

Comment:
### OPTIONAL 

- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿 
- See suggestions how to fix this error [here](https://www.google.com/search?q=unexpected+use+of+file+extension+%22jsx%22+for&oq=&aqs=chrome.0.35i39i362l8.322964j0j7&sourceid=chrome&ie=UTF-8)